dev_guide/builds.adoc

```diff
@@ -1064,6 +1063,9 @@ $ oc secrets add serviceaccount/builder secrets/scmsecret
 ----
 ====
 
+NOTE: Depending on the `*limitSecretReferences*`, `*serviceAccountConfig*` section
```

Comment:
Yeah, after I dig into the topic I started feeling the same way. But on the other hand if you go through the steps they seem so artificial. One other though would be to explain this step, why this add is needed, would that make sense?